.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -36,7 +36,7 @@ conda activate "${env_dir}"
 
 
 # 3. Install minimal build tools
-pip --quiet install cmake ninja
```

Comment:
Maybe we need an equivalent change in windows setup script.

It seems like an old version of CMake can cause dynamic link issue

https://stackoverflow.com/a/37146557